pkg/cli/admin/admin.go

```diff
@@ -57,7 +55,7 @@ func NewCommandAdmin(f kcmdutil.Factory, streams genericiooptions.IOStreams) *co
 	cmds := &cobra.Command{
 		Use:   "adm",
 		Short: "Tools for managing a cluster",
-		Long:  fmt.Sprintf(adminLong),
+		Long:  string(adminLong),
```

Comment:
Why can we simply use `adminLong`?

Reply:
You are doing review so fast that you end up reviewing retarded code I haven't checked properly yet 😅 Anyway, I should do better work for the first draft. You are right, of course, the cast is not needed.